sherpa/plot/tests/test_bokeh.py

```diff
@@ -53,3 +58,59 @@ def test_bokeh_delchi(caplog, clean_ui):
     newback = bokehbackend.BokehBackend()
     with plot.TemporaryPlottingBackend(newback):
         ui.plot_fit_delchi()
+
+
+def test_bokeh_specific_option_does_not_error():
+    """Test that a bokeh-specific option does not error out.
+
+    """
+    bokehbackend = pytest.importorskip("sherpa.plot.bokeh_backend")
+    # previous line guarantees that bokeh is installed
+    from bokeh.embed import file_html
+    from bokeh.resources import CDN
+
+    data = Data1DInt('tst', np.asarray([1, 2, 3]), np.asarray([1, 2, 3]) + 1,
+                      np.asarray([10, 12, 10.5]))
+    pl = DataPlot()
```

Comment:
```suggestion
    pl = DataHistogramPlot()
```

and then also importing it above.

The naming of our plot classes, and what they are meant to support, is lost to the mist of time. 

We also allow users to populate `DataPlot` with a "integrated" data set like `Data1DInt` so maybe this change is not worth it, but I thought I'd check.

Reply:
Either will work for this test. The point is that the plot commands are passed through to `BokehBackend.plot` and that will happen with both a `DataPlot` and the `DataHistogramPlot`. I'm using `DataPlot` and `ScatterPlot` for bokeh and `DataPlot`, `ModelHistrogramPLot` and `Contour` for matplotlib, just to get some variety without writing a test for evert plot class we have (since the all pass it through in the same way to the same underlying function). 
I could switch one of the `DataPlot` uses to a `DataHistrogramPlot` for good measure, but it doesn't really matter.